src/ImageSharp/Formats/Bmp/BmpInfoHeader.cs

```diff
@@ -66,6 +66,16 @@ internal struct BmpInfoHeader
         /// </summary>
         public const int HeaderSizeSize = 4;
 
+        /// <summary>
+        /// Maximum dimensions of a bitmap with or height: 2**31 - 1, since width and height are int32
+        /// </summary>
+        private const int MaximumBmpDimension = 2147483647;
```

Comment:
This is not being used?

Reply:
no, it's no longer used, it's now removed.